verilog/tools/ls/verilog_ls.cc

```diff
@@ -129,6 +132,8 @@ int main(int argc, char *argv[]) {
   // It registers callbacks to receive the relevant events on the dispatcher.
   BufferCollection buffers(&dispatcher);
 
+  verilog::SymbolTableHandler symboltablehandler;
```

Comment:
the `symboltablehandler` gets called in the dispatcher. You want to avoid the dispatcher calling an out-of-scope object so its lifetime needs to be longer than that, i.e. you need to declare it before.

Reply:
I fixed it now.